### DISCUSSION TITLE
Conflicts resolution

### DISCUSSION THREAD

**Discussion by Arslan007** [Azure/azure-mobile-apps#750] [Q&A]
We are using sync framework in MAUI hybrid and in our case data can be updated on any platform and conflicts can occur that we are handling already. There is one special scenario lets say data is updated on Web and windows app and conflicts occurred on windows app and we displayed conflict screen to choose which copy you want remote or local and if user kills the app at this moment. Sync framework always automatically updates with remote copy. I believe that is handled on app closing automatically by syncframework internally. Is there anyway we can keep local copy in this scenario.

Thanks

**Comment by adrianhall** [comment]
Sync framework should not be automatically updating on conflict.  When a conflict occurs, an error is placed on an errors queue - nothing has been done to the server side.  The request is still enqueued and awaiting disposition.  When your application re-opens, the errors queue is cleared, but the queued entry is still there, so it should be re-submitted when your windows app initiates a sync again.

If that isn't happening, create an xUnit test to simulate the case and submit an issue showing it is failing in that case.  Without a reproducable case, I don't think I'll be able to fix it.
